src/Orckestra.Composer.Website/App_Data/Razor/Composer/Product/Specifications.cshtml

```diff
@@ -17,54 +17,105 @@
     {
         return Model.Variants == null || Model.Variants.Count == 0;
     }
+
+    public bool HasSpecifications()
+    {
+        if (IsWithoutVariants())
+        {
+            return Model.Specifications.Groups != null || Model.Specifications.Groups.Count != 0;
+        }
+        else
+        {
+            return HasVariantSpecifications() || Model.Specifications.Groups != null || Model.Specifications.Groups.Count != 0;
+        }
+    }
+
+    public bool HasVariantSpecifications()
+    {
+        var variantsSpecs = Model.Variants.Where(v => v.Specifications != null);
```

Comment:
can be simplified 
var hasSpecs = Model.Variants.Where(v => v.Specifications != null && v.Specifications.Groups!== null && spec.Specifications.Groups.Count != 0).Any();

Reply:
Thanks !